### PR TITLE
Mitigate Select component Formation problems

### PR DESCRIPTION
**Pull request by Andrew565**
## Chromatic

https://{{head.ref}}--60f9b557105290003b387cd5.chromatic.com

---
## Configuring this pull request
- [X] Link to any related issues in the description so they can be cross-referenced.
- [X] Under **Testing done**, be specific about how this update was tested. 
    - Examples: Storybook, Chrome, Browserstack, Mobile/Responsive, etc.
- [X] Add the appropriate version patch label (`major`, `minor`, `patch`, or `ignore-for-release`). 

## Description
Closes #[1568](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1568)

## Testing done
Tested on Storybook using a desktop browser, and in the DS Playground using a desktop browser.

## Screenshots
Screenshot 2023-03-01 at 12 47 37 PM

## Acceptance criteria
- [X] Renders correctly

### CODE REVIEW

File: packages/web-components/src/global/_variables.scss [hunk 1]
```diff
@@ -84,4 +84,4 @@ $units-neg-7: -5.6rem;
 $units-neg-8: -6.4rem;
 $units-neg-9: -7.2rem;
 $units-neg-10: -8rem;
-$units-neg-15: -12rem;
\ No newline at end of file
+$units-neg-15: -12rem;
```
Comment (Andrew565) [inline packages/web-components/src/global/_variables.scss:87]: please ignore

File: packages/web-components/src/components/va-radio-option/va-radio-option.scss [hunk 2]
```diff
@@ -71,13 +72,13 @@
   margin-top: 5px;
 }
 
-:host(:not([uswds])[tile="true"]) label {
+:host(:not([uswds])[tile='true']) label {
```
Comment (Andrew565) [inline packages/web-components/src/components/va-radio-option/va-radio-option.scss:75]: No winning for trying somedays. Please ignore.

**Comment by jamigibbs** [comment]
Can these first two items on the [PR configuration checklist](https://github.com/department-of-veterans-affairs/component-library/blob/main/.github/PULL_REQUEST_TEMPLATE.md) be completed?

Screenshot 2023-03-01 at 12 20 16 PM

**Review by jamigibbs — APPROVED** [review]
I'm not sure why the Chromatic link didn't work but I confirmed locally. 👍

**Review by jamigibbs — APPROVED** [review]
This looks good. You'll just need to add back these style overrides before merging:

Screenshot 2023-03-03 at 9 14 41 AM

**Pull request merged**